linux_os/guide/system/network/network-ipv6/disabling_ipv6/kernel_module_ipv6_option_disabled/tests/module_and_sysctl_disabled.pass.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+# platform = Red Hat Enterprise Linux 7
+
+echo "options ipv6 disable=1" > /etc/modprobe.d/ipv6.conf
```

Comment:
The test is named module_and_sysctl_disabled... but I don't see anything about sysctl in this test. Is the sysctl option disabled by default?

Reply:
Nope, test renamed.